src/App.js

```diff
@@ -1,29 +1,28 @@
 import React from 'react';
-
 import './App.css';
 
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
-const App = () => (
-  <div className="App">
-    <h1>Static list of posts</h1>
+import PostList from './components/PostList/PostList';
 
-    <p>
-      <span>posts: </span>
-      {posts.length}
-    </p>
+const getPostsWithUserAndComments = (
+  postList,
+  userList,
+  commentList
+) => postList.map(post => ({
```

Comment:
better to open parentheses here and place the function body on the new line

Reply:
Iryna, do you mean like this? Pushed and deployed.
![image](https://user-images.githubusercontent.com/32244670/64686662-b1368f00-d491-11e9-9d56-25df25cf2cd7.png)